sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/issue/NewIssue.java

```diff
@@ -74,6 +78,18 @@ public interface NewIssue {
    */
   NewIssue addFlow(Iterable<NewIssueLocation> flowLocations);
 
+  /**
+   * Register a flow for this issue. A flow is an ordered list of issue locations that help to understand the issue.
```

Comment:
Sidenote: I think in JavaDoc we should be using third-person singular verb - so Register**s** instead of Register - This is how Java's, Apache Commons, Gson etc.  JavaDocs are written. Although I am sticking here to what already exists.

Reply:
I agree :+1: